From weiyi.huang@gmail.com  Mon Oct 26 15:07:27 2009
From: Huang Weiyi <weiyi.huang@gmail.com>
Date: Thu, 17 Sep 2009 21:25:49 +0800
Subject: Staging: vme: remove unused #include <linux/version.h>
To: gregkh@suse.de
Cc: devel@driverdev.osuosl.org, Huang Weiyi <weiyi.huang@gmail.com>
Message-ID: <1253193949-3368-1-git-send-email-weiyi.huang@gmail.com>


Remove unused #include <linux/version.h>('s) in
  drivers/staging/vme/bridges/vme_ca91cx42.c
  drivers/staging/vme/bridges/vme_tsi148.c
  drivers/staging/vme/devices/vme_user.c
  drivers/staging/vme/vme.c

Signed-off-by: Huang Weiyi <weiyi.huang@gmail.com>
Acked-by: Martyn Welch <martyn.welch@gefanuc.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/vme/bridges/vme_ca91cx42.c |    1 -
 drivers/staging/vme/bridges/vme_tsi148.c   |    1 -
 drivers/staging/vme/devices/vme_user.c     |    1 -
 drivers/staging/vme/vme.c                  |    1 -
 4 files changed, 4 deletions(-)

--- a/drivers/staging/vme/bridges/vme_ca91cx42.c
+++ b/drivers/staging/vme/bridges/vme_ca91cx42.c
@@ -15,7 +15,6 @@
  * option) any later version.
  */
 
-#include <linux/version.h>
 #include <linux/module.h>
 #include <linux/mm.h>
 #include <linux/types.h>
--- a/drivers/staging/vme/bridges/vme_tsi148.c
+++ b/drivers/staging/vme/bridges/vme_tsi148.c
@@ -13,7 +13,6 @@
  * option) any later version.
  */
 
-#include <linux/version.h>
 #include <linux/module.h>
 #include <linux/moduleparam.h>
 #include <linux/mm.h>
--- a/drivers/staging/vme/devices/vme_user.c
+++ b/drivers/staging/vme/devices/vme_user.c
@@ -31,7 +31,6 @@
 #include <linux/spinlock.h>
 #include <linux/syscalls.h>
 #include <linux/types.h>
-#include <linux/version.h>
 
 #include <asm/io.h>
 #include <asm/uaccess.h>
--- a/drivers/staging/vme/vme.c
+++ b/drivers/staging/vme/vme.c
@@ -13,7 +13,6 @@
  * option) any later version.
  */
 
-#include <linux/version.h>
 #include <linux/module.h>
 #include <linux/moduleparam.h>
 #include <linux/mm.h>
